projboard/views.py

```diff
@@ -1,5 +1,19 @@
 from django.shortcuts import render
+from .models.article import Article
 
 
 def board(request):
     return render(request, 'projboard/board.html', {})
+
+
+def search(request):
+    title = ""
+    articles = []
+    if request.method == 'POST':
+        title = request.POST['title']
+        if title != '':
+            articles = Article.search_by_title(title)
+
+    return render(request, 'projboard/searchArticle/search_article.html', {'articles': articles,
+                                                                           'numArticles': len(articles),
```

Comment:
Consider different grammer as per py guide

Reply:
sorry , i didn't understand , can you explain again ?

Reply:
Please refer this 
https://peps.python.org/pep-0008/#names-to-avoid

Reply:
no change here

Reply:
Added comment in the template see naming conventions